app/views/pages/wish_list/create.liquid

```diff
@@ -0,0 +1,12 @@
+---
+method: post
+---
+{% liquid
+  function_rc object = 'lib/commands/wish_list/create/build', item_id: context.params.id, user_id: context.params.user_id
```

Comment:
`function_rc` is deprecated and replaced with `function` tag

Reply:
Thanks for the info, changed `function_rc` calls to `function` in the next commit